src/control/lib/daos/pool_property.go

```diff
@@ -325,7 +325,7 @@ func PoolProperties() PoolPropertyMap {
 		"scrub_freq": {
 			Property: PoolProperty{
 				Number:      PoolPropertyScrubFreq,
-				Description: "Checksum scrubbing frequency",
+				Description: "Scrubbing frequency",
 				valueHandler: func(s string) (*PoolPropertyValue, error) {
 					rbErr := errors.Errorf("invalid Scrubbing Frequency value %s", s)
```

Comment:
I know that you didn't add this originally, but since you're making changes, it would be a nicer UX to accept and display human-comprehensible frequency values. Pushed a PR that you could merge if you agree: https://github.com/daos-stack/daos/pull/14650